### ISSUE TITLE
Container restart resets config file permissions

### ISSUE THREAD

**Issue by rounakdatta** [NathanVaughn/webtrees-docker#89]
```
Fatal error: Uncaught ErrorException: parse_ini_file(/var/www/webtrees/data/config.ini.php)
```

I get this error every time the webtrees container gets restarted. This blocks the application entirely. When I look at the file permissions of the `config.ini.php` file,
```
---------- 1 www-data www-data 231 Mar 18 15:12 config.ini.php
```

Currently I fix it everytime by manually running
```
chmod 700 config.ini.php
```

Is this a configuration issue on my side, or an idempotency issue in the Dockerfile? I believe we needn't generate/edit the config.ini.php file everytime.

**Comment by NathanVaughn** [comment]
This should be resolved now with the latest push of the `:2.0.21` tag and new entrypoint script. Feel free to re-open if it continues to be a problem.

**Comment by rounakdatta** [comment]
Thanks for such a quick turnaround, I took a pull of `:2.0.21` and indeed the problem is fixed!
